lib/plugins/encrypted_parameters/kms_key.rb

```diff
@@ -11,12 +13,27 @@ def initialize(arn)
         end
 
         def self.create
-          resp = Aws::KMS::Client.new.create_key
+          standard_tags=KmsKey.stack_tags
+          resp = Aws::KMS::Client.new.create_key({
+            tags: standard_tags, # An array of tags.
+          })
           arn = resp.key_metadata.arn
-
           new(arn)
         end
 
+        def update
+          standard_tags=KmsKey.stack_tags
+          resp = @kms_client.tag_resource({
+            key_id: @arn, # arn of the CMK being tagged
+            tags: standard_tags, # An array of tags.
+          })
+        end
+
+        def self.stack_tags
```

Comment:
https://railsware.com/blog/better-ruby-choosing-convention-for-class-methods-definition/, 
how about you just follow this convention.
Put al class methods in the class << self block. It will be more intuitive.